app/modules/individuals/resources.py

```diff
@@ -18,12 +19,35 @@
 
 from . import parameters, schemas
 from .models import Individual
-
+from app.modules.encounters.models import Encounter
```

Comment:
There have been circular dependency issues with model imports before. I think the general plan is it's safer to just import these in teh functions they're used in

Reply:
It just irks me whenever I have to import something for multiple functions on the same level.. Ah well, that's not even the case here.